assets/items/pistol.item

```diff
@@ -0,0 +1,13 @@
+{
+	"name": "Plasma Pistol",
+	"content": {
+		"model": {
+			"Hand": {
+				"Path": "models/pistol.glb"
```

Comment:
All paths in config files point to the assets folder, because they are loaded via bevy, which loads assets from the asset folder. While this might be marked better, it is in line with how bevy handles assets.